spec/features/shelters/cannot_delete_with_approved_pets_spec.rb

```diff
@@ -43,4 +52,20 @@
     visit "/shelters"
     expect(page).to_not have_content("Delete #{@shelter_1.name}")
   end
+
+  it "can delete a shelter when no pets have a pending status" do
+    visit "/shelters"
+      within("#shelter-#{@shelter_2.id}") do
+        click_link "Delete #{@shelter_2.name}"
+      end
+
+      expect(page).to_not have_content(@shelter_2.address)
+      expect(page).to_not have_content(@shelter_2.city)
+      expect(page).to_not have_content(@shelter_2.state)
+      expect(page).to_not have_content(@shelter_2.zip)
+
+      visit "/pets"
```

Comment:
Originally wanted to use a within but it was not possible here since the pet's id was deleted. Could see an error if a future flash message declares the name of the pet that was deleted.